command.ts

```diff
@@ -792,145 +822,87 @@ XRANGE somestream - +
-  acl_genpass(parameter?: number): Promise<Status>;
-  acl_getuser(parameter: string): Promise<BulkString[]>;
+  acl_cat(categoryname?: string): Promise<BulkString[]>;
+  acl_deluser(...usernames: string[]): Promise<Integer>;
+  acl_genpass(bits?: number): Promise<BulkString>;
+  acl_getuser(username: string): Promise<(BulkString | BulkString[])[]>;
   acl_help(): Promise<BulkString[]>;
   acl_list(): Promise<BulkString[]>;
   acl_load(): Promise<Status>;
-  acl_log(parameter: string | number): Promise<Status | BulkString[]>;
+  acl_log(count: number): Promise<BulkString[]>;
+  acl_log(mode: "RESET"): Promise<Status>;
   acl_save(): Promise<Status>;
-  acl_setuser(username: string, rule: string): Promise<Status>;
+  acl_setuser(username: string, ...rules: string[]): Promise<Status>;
   acl_users(): Promise<BulkString[]>;
-  acl_whoami(): Promise<Status>;
+  acl_whoami(): Promise<BulkString>;
   bgrewriteaof(): Promise<Status>;
   bgsave(): Promise<Status>;
-  // client //
   command(): Promise<
-    [BulkString, Integer, BulkString[], Integer, Integer, Integer]
+    [BulkString, Integer, BulkString[], Integer, Integer, Integer][]
   >;
   command_count(): Promise<Integer>;
   command_getkeys(): Promise<BulkString[]>;
-  command_info(...command_names: string[]): Promise<[
-    [
-      BulkString,
-      Integer,
-      BulkString[],
-      Integer,
-      Integer,
-      Integer,
-      [BulkString[]],
-    ] | BulkNil,
-  ]>;
+  command_info(
+    ...command_names: string[]
+  ): Promise<
+    ([BulkString, Integer, BulkString[], Integer, Integer, Integer] | BulkNil)[]
+  >;
   config_get(parameter: string): Promise<BulkString[]>;
+  config_resetstat(): Promise<Status>;
   config_rewrite(): Promise<Status>;
   config_set(parameter: string, value: string): Promise<Status>;
-  config_resetstat(): Promise<Status>;
   dbsize(): Promise<Integer>;
   debug_object(key: string): Promise<Status>;
   debug_segfault(): Promise<Status>;
   flushall(async?: boolean): Promise<Status>;
   flushdb(async?: boolean): Promise<Status>;
-  info(section?: string): Promise<Status>;
+  info(section?: string): Promise<BulkString>;
   lastsave(): Promise<Integer>;
-  memory_doctor(): Promise<Status>;
+  memory_doctor(): Promise<BulkString>;
   memory_help(): Promise<BulkString[]>;
-  memory_malloc_stats(): Promise<Status>;
+  memory_malloc_stats(): Promise<BulkString>;
   memory_purge(): Promise<Status>;
   memory_stats(): Promise<ConditionalArray>;
-  memory_usage(
-    key: string,
-    opts?: {
-      samples?: number;
-    },
-  ): Promise<Integer>;
+  memory_usage(key: string, opts?: { samples?: number }): Promise<Integer>;
   module_list(): Promise<BulkString[]>;
-  module_load(path: string, args: string): Promise<Status>;
+  module_load(path: string, ...args: string[]): Promise<Status>;
   module_unload(name: string): Promise<Status>;
   monitor(): void;
+  replicaof(host: string, port: number): Promise<Status>;
+  replicaof_none(): Promise<Status>;
```

Comment:
This name is nice, but for consistency I want to match the convention with other methods.
```suggestion
  replicaof_no_one(): Promise<Status>;
```